README.md

```diff
@@ -1,3 +1,14 @@
 # FixMe
 
-[![Build Status](https://github.com/CasBex/FixMe.jl/actions/workflows/CI.yml/badge.svg?branch=main)](https://github.com/CasBex/FixMe.jl/actions/workflows/CI.yml?query=branch%3Amain)
+[![Stable Documentation](https://img.shields.io/badge/docs-stable-blue.svg)](https://CasBex.github.io/FixMe.jl/stable)
+[![In development documentation](https://img.shields.io/badge/docs-dev-blue.svg)](https://CasBex.github.io/FixMe.jl/dev)
+[![Build Status](https://github.com/CasBex/FixMe.jl/workflows/Test/badge.svg)](https://github.com/CasBex/FixMe.jl/actions)
+[![Test workflow status](https://github.com/CasBex/FixMe.jl/actions/workflows/Test.yml/badge.svg?branch=main)](https://github.com/CasBex/FixMe.jl/actions/workflows/Test.yml?query=branch%3Amain)
+[![Lint workflow Status](https://github.com/CasBex/FixMe.jl/actions/workflows/Lint.yml/badge.svg?branch=main)](https://github.com/CasBex/FixMe.jl/actions/workflows/Lint.yml?query=branch%3Amain)
+[![Docs workflow Status](https://github.com/CasBex/FixMe.jl/actions/workflows/Docs.yml/badge.svg?branch=main)](https://github.com/CasBex/FixMe.jl/actions/workflows/Docs.yml?query=branch%3Amain)
+[![Coverage](https://codecov.io/gh/CasBex/FixMe.jl/branch/main/graph/badge.svg)](https://codecov.io/gh/CasBex/FixMe.jl)
+[![DOI](https://zenodo.org/badge/DOI/FIXME)](https://doi.org/FIXME)
+
```

Comment:
Not all badges seem to work. Does this have to do with the fact I used PkgTemplates initially?

Reply:
No, it's because they don't exist yet. You need 3 things to make them all pass:

- Merge the PR and run the workflows (it will get to this state: https://github.com/abelsiqueira/BestieMinimum.jl)
- Enable code coverage with Codecov (https://abelsiqueira.com/BestieTemplate.jl/stable/10-full-guide/#Enable-Codecov-for-code-coverage)
- DOI is only fixed when you create a DOI (either manually or via GitHub integration) and then manually fix the DOI number (https://abelsiqueira.com/BestieTemplate.jl/stable/10-full-guide/#CITATION.cff-and-Zenodo-deposition).

Reply:
Ok, in that case I suggest removing the DOI and leaving the rest.

Reply:
I suggest leaving the code coverage badge in, and we can let mentees try to get the code coverage working in one of the assignments.